sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubAsyncClient.java

```diff
@@ -63,31 +52,25 @@
     public static final String DEFAULT_CONSUMER_GROUP_NAME = "$Default";
 
     private static final String RECEIVER_ENTITY_PATH_FORMAT = "%s/ConsumerGroups/%s/Partitions/%s";
-    private static final String SENDER_ENTITY_PATH_FORMAT = "%s/Partitions/%s";
 
     private final ClientLogger logger = new ClientLogger(EventHubAsyncClient.class);
     private final MessageSerializer messageSerializer;
-    private final Mono<EventHubConnection> connectionMono;
+    private final EventHubLinkProvider linkProvider;
```

Comment:
EventHubAsyncClient should now be package-private.

Reply:
This is going to be addressed in another PR with the rename of EventHubClient -> EventHubConnection (and all its methods package private).